core/sys/energest.h

```diff
@@ -105,6 +105,21 @@ extern energest_t energest_leveldevice_current_leveltime[ENERGEST_CONF_LEVELDEVI
 										energest_current_time[type]); \
 										energest_current_mode[type] = 0; \
                                        } while(0)
+
+#define ENERGEST_SWITCH(type_off, type_on) do { \
+                                             rtimer_clock_t now = RTIMER_NOW(); \
```

Comment:
There is a chance this will collide with other local variables that happen to be called `now` - a better name might be `energest_local_variable_now` or something similar (which has a much lower risk of collision).